entries/codes/starlink.yaml

```
@@ -0,0 +1,24 @@
+name: Starlink
+category: codes
+tags: 
+text: The Starlink software (Currie et al. 2014) is currently supported by the East Asian Observatory.
+latex: The Starlink software \citep{Currie_2014} is currently supported by the East Asian Observatory.
+url: https://starlink.eao.hawaii.edu/starlink
+dependencies: 
+bibtex: >
+  @INPROCEEDINGS{Currie_2014,
+       author = {{Currie}, M.~J. and {Berry}, D.~S. and {Jenness}, T. and {Gibb}, A.~G. and {Bell}, G.~S. and {Draper}, P.~W.},
+        title = "{Starlink Software in 2013}",
+    booktitle = {Astronomical Data Analysis Software and Systems XXIII},
+         year = 2014,
+       editor = {{Manset}, N. and {Forshay}, P.},
+       series = {Astronomical Society of the Pacific Conference Series},
+       volume = {485},
+        month = may,
+        pages = {391},
+       adsurl = {https://ui.adsabs.harvard.edu/abs/2014ASPC..485..391C},
+      adsnote = {Provided by the SAO/NASA Astrophysics Data System}
+}
+
+
+
```

```suggestion
  }
```
Stupid white sapce requirements ;-)